mt-kahypar/partition/refinement/fm/strategies/km1_gains.h

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+ * This file is part of MT-KaHyPar.
+ *
+ * Copyright (C) 2020 Lars Gottesbüren <lars.gottesbueren@kit.edu>
+ *
+ * KaHyPar is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * KaHyPar is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with KaHyPar.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ ******************************************************************************/
+
+#pragma once
+
+#include "mt-kahypar/datastructures/hypergraph_common.h"
+
+namespace mt_kahypar {
```

Comment:
we have three different files now doing the same thing: km1_gains.h, gain_computation_policy.h and gain_policy.h => we should somehow summarize them in one file or at least km1_gains.h and gain_policy.h